autoheal.yml

```diff
@@ -34,17 +34,41 @@ awx:
   proxy: http://my-proxy.example.com:3128
 
   #
-  # Reference to the Kubernetes secret that contains the user name and password
-  # to use to connect to the AWX API.
+  # The user name and password used to connect to the AWX API.
+  #
+  credentials:
```

Comment:
What happens if both sections (`credentials` and `credentialsRef`) exist? Which one takes over?

I think that developer-oriented configuration should commented out in this file
```
# credentials:
#    username: my-user
#    password: my-password
```

Reply:
Done in the next version of the pull request.